README.md

```diff
@@ -1,10 +1,11 @@
 # 🖼️ image.nvim
 
-> **Warning**
->\
-> This is very much a **work in progress**, **there are many bugs**, and there's lots to improve, but... we're getting there!
+This plugin attempts to add image support to Neovim.
 
-**image.nvim** is an attempt to add image support to Neovim.
+It works wonderfully with Kitty + Tmux, and it handles painful things like rendering an image
+at a given position in a buffer, scrolling, windows, etc.
```

Comment:
I read this and went, wait really? there are so many bugs when I scroll.  So I launch kitty to make sure it's not wezterm.

Turns out wezterm's kitty graphics protocol implementation < kitty's implementation. perhaps unsurprisingly.

Reply:
Yep, it's pretty cool. There are still some glitches with multiple windows, but it's generally snappy when scrolling!


https://github.com/3rd/image.nvim/assets/59587503/36335554-5574-45e4-9eb2-7335783f19c6

Reply:
It only glitches when the same buffer is loaded in two panes and things like folding happens in one of them, because the virtual space is buffer bound and can't be per-window.